whois-api/src/main/java/net/ripe/db/whois/api/rest/WhoisSearchService.java

```diff
@@ -77,7 +75,6 @@
             // flags that are covered by path/query params or other rest calls
             TEMPLATE,
             VERBOSE,
-            CLIENT,
```

Comment:
Is "-V" now allowed in the "flags" query parameter (in addition to the new client-attribute parameter?)

Reply:
Ahh. Good catch. I forgot to add it again. Once I started using query parameter